api/src/main/webapp/WEB-INF/cron.xml

```diff
@@ -30,4 +30,10 @@
     <description>Daily sync of 2FA set up status (via Google Directory Service)</description>
     <schedule>every 24 hours</schedule>
   </cron>
+  <cron>
+    <url>/v1/cron/bufferBillingProject</url>
+    <target>api</target>
+    <description>If the AoU Billing Project buffer is not full, create a firecloud billing project and add one</description>
+    <schedule>every 5 seconds</schedule>
```

Comment:
This may be too rapid; I think 1 per minute might suffice (we can tune this later if needed)